Automattic/wp-calypso
# WPCOM SSH: Add reauth form to hosting config page to display ssh keys when cookie expire

#### Issue
SSH requires 2fa and the cookie that is sent `twostep_auth` can expire. Visiting hosting config when this is expired result in the SSH section loading and then throw an error in the console. This prevent the user from seeing their keys and attaching more keys.

![image](https://user-images.githubusercontent.com/47489215/197907891-0093196f-6c2c-4e4e-b12d-d862cac9ee0d.png)

#### Steps to Repro

1. Delete `twostep_auth` 
2. Go to hosting config page then observe the error in the console

#### Proposed Changes

* Add reauth popup to hosting config page so the user can reauthenticate when cookie expire so their ssh keys will be displayed
* Listen for change events from `twoStepAuthenticaton` and render `Sshkeys` component when successful.

#### Testing Instructions/

1. Delete `twostep_auth` 
2. Go to hosting config page, the reauth form should popup
3. Reauth then verify you can now see your ssh keys 



*

#### Pre-merge Checklist






Related to #
